src/resolvers/Mutation/acceptAdmin.ts

```diff
@@ -54,5 +55,11 @@ export const acceptAdmin: MutationResolvers["acceptAdmin"] = async (
     }
   );
 
+  await storeTransaction(
+    context.userId,
+    TRANSACTION_LOG_TYPES.UPDATE,
+    "User",
+    `User:${args.id} accepted as admin`
+  );
```

Comment:
This is not a good approach at all. 
- Logging is backend-specific. If we are awaiting the logging operation in queries, it increases the time our sever takes to respond to each request. Logging should be asynchronous. 
- Can you research a better way of doing this? Adding a log to each file for each transaction is a  lot of work, and difficult to maintain in the upcoming PRs. There should be some middleware or existing package that we can leverage to log all the database changes. [Reference](https://www.mongodb.com/docs/manual/reference/log-messages/)
- These user messages are helpful in debugging! This change is in the right direction, but Im really skeptical about the implementation.
- In the next commit, please make changes according to your new approach in only a couple of files to they are easy to review. Once you get your method approved, you can make the changes to all the files. This would save a lot of effort on you part as well.